website/content/en/docs/tasks/set-resource-limits.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Set Resource Limits"
+linkTitle: "Set Resource Limits"
+weight: 10
+---
+
+Karpenter automatically provisions instances from the cloud provider. This often incurs hard costs. To control resource utilization and cluster size, use resource limits.
```

Comment:
We should also call this out in our provisioner doc - https://github.com/aws/karpenter/blob/main/website/content/en/docs/provisioner.md